### PR TITLE
WR#249863: Fixed  #4. Converting raw English lang strings to Moodle l…

### LINKED ISSUE #4: Hard-coded English texts in index.php
E.g.

```
<table>
<tr><td>Bot user                        <td><?php echo $boterror   ? $boterror : 'Good' ?>
<tr><td>Current crawl started at        <td><?php echo $crawlstart ? userdate( $crawlstart) : 'Never run'      ?>
<tr><td>Last crawl ended at             <td><?php echo $crawlend   ? userdate( $crawlend)   : 'Never finished' ?>
<tr><td>Last crawl process              <td><?php echo $crawltick  ? userdate( $crawltick)  : '-'              ?>
```

### PR DESCRIPTION
**Pull request by suankan**
…ang strings. Refactoring drawing HTML table in index.php with html_table().

Please note, this one will require manual merge resolution when applying to master.

### CODE REVIEW

**Comment by brendanheywood** [comment]
hey @suankan please rebase this onto master to fix the conflicts and I'll review tomorrow

File: index.php [hunk 1]
```diff
@@ -60,56 +59,50 @@
...
-        <td><a href="report.php?report=oversize"><?php echo $oversize; ?></a></td>
-    </tr>
-</table>
+$table = new html_table();
+$table->head = array(get_string('robotstatus', 'local_linkchecker_robot'));
+$table->headspan = array(2, 1);
+$table->data = array(
+    array(
+        get_string('botuser', 'local_linkchecker_robot'),
+        $boterror ? $boterror : 'Good'
```
Comment (brendanheywood) [inline index.php]: you have fixed this already, was there more commits you nee to squash into this branch?

File: lang/en/local_linkchecker_robot.php [hunk 2]
```diff
@@ -88,4 +90,6 @@
 $string['slowurl'] = 'Slow URL';
 $string['found'] = 'Found';
 $string['localplugins'] = 'Local Plugins';
-$string['numberurlsfound'] = 'Found {$a->reports_number} {$a->repoprt_type}  URLs';
\ No newline at end of file
+$string['numberurlsfound'] = 'Found {$a->reports_number} {$a->repoprt_type}  URLs';
+$string['robotstatus'] = 'Robot status';
+$string['autocreate'] = 'Auto create';
```
Comment (brendanheywood) [inline lang/en/local_linkchecker_robot.php]: make sure all files end with eol - if need fix you ide settings

File: index.php (hunk 1 above)
Comment (suankan) [inline index.php]: Initially I was just fixing the code style and fixed the HTML tags in a certain commit. And yesterday I've put that commit into the branch code_style_fixes. 
And looks like later on we have decided to redo rendering this HTML table by mean of html_table() and that went to another commit which I yesterday put into this branch "issue4_eng_lang_text".

So this is the final change in this piece of code and it should remain the same during our next updates.

File: lang/en/local_linkchecker_robot.php (hunk 2 above)
Comment (suankan) [inline lang/en/local_linkchecker_robot.php]: I've enabled strip trailing spaces in my phpstorm.

Comment (brendanheywood) [inline lang/en/local_linkchecker_robot.php]: still missing a new line here, please add it in

**Pull request merged**